app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+list_of_variables = [lucky_number, pi, one_is_a_prime_number,
+                     name, my_favourite_films, profile_info,
+                     marks, collection_of_coins]
+list_of_mutable = []
+list_of_immutable = []
+sorted_variables = {"mutable": list_of_mutable,
+                    "immutable": list_of_immutable}
+for element in list_of_variables:
+    if isinstance(element, (list, set, dict)):
+        list_of_mutable.append(element)
+    else:
+        list_of_immutable.append(element)
+
+sorted_variables_manual = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
No additional logic is required - just pass the variables to the correct list in the dictionary.